infra/build-pipelines.yml

```diff
@@ -1,32 +1,41 @@
-# ASP.NET Core (.NET Framework)
-# Build and test ASP.NET Core projects targeting the full .NET Framework.
-# Add steps that publish symbols, save build artifacts, and more:
-# https://docs.microsoft.com/azure/devops/pipelines/languages/dotnet-core
-name: 'Unknown until run'
+# Build the app and modules
+name: 'Known during build'
 
 parameters:
-- name: overwrite_existing_tag
+- name: version_number
+  displayName: 'Version Number (e.g. 1.0)'
+  type: string
+  default: '1.0'
+- name: add_tag
+  displayName: 'Add build tag to repo'
   type: boolean
   default: true
+- name: target_os
+  displayName: 'Target architecture'
+  type: string
+  default: 'win-x64' 
+  values:
+  - win-x64
+  - linux-x64
```

Comment:
hey @simon-wang-gc ! did you mean to have this default to `win-x64`?

Reply:
Eventually it should be linux. I just have not been able to make it work on Linux yet but will get it done soon.